config/serverless.yml

```diff
@@ -17,3 +17,6 @@ xpack.license_management.enabled: false
 # Other disabled plugins
 #xpack.canvas.enabled: false #only disabable in dev-mode
 xpack.reporting.enabled: false
+
+# Enforce restring access to internal APIs see https://github.com/elastic/kibana/issues/151940
+# server.restrictInternalApis: true
```

Comment:
Added here for intended purpose. Once all intra-stack components and Kibana browser-side code not using Core's http service are updated, we can enable the restriction.